yt_dlp/extractor/ard.py

```diff
@@ -640,6 +658,12 @@ def _real_extract(self, url):
             'description': description,
             'timestamp': unified_timestamp(player_page.get('broadcastedOn')),
             'series': try_get(player_page, lambda x: x['show']['title']),
+            'thumbnail': try_get(player_page, lambda x: x['image']['src'].replace('{width}', '1280')),
```

Comment:
The SEO tags were using 1280, so that seems to be some default number decided by ard. But also bigger numbers work, I am just not sure if they are actually higher quality or just scaled up. The maximum resolution I could get in my test is 4096, higher numbers fell back on 4096. It is a strange number for image width though

Reply:
4096 = 4 * 1024 = 1 << 12

That doesnt seem that odd.

You could also use the `thumbnails` key and return multiple widths, one being max and size and one the default

Reply:
Removing the `w` param completely gives a 540p which I assume is the actual thumbnail.

Reply:
I see, in that case I will just remove the w parameter altogether. But I will also check with a friend if he can tell what is happening with the bigger images, I suppose it might be some smoothing going on

Reply:
In fact, the query parameters can be stripped completely:
`https://api.ardmediathek.de/image-service/images/urn:ard:image:efa186f7b0054957`

Reply:
Additional evidence, the other test case gives a default resolution of 1080p: https://api.ardmediathek.de/image-service/images/urn:ard:image:efa186f7b0054957?&ch=b09581b1bc37d0eb